src/read.c

```diff
@@ -1759,6 +1760,18 @@ static avifBool avifParseColourInformationBox(avifProperty * prop, uint64_t rawO
     return AVIF_TRUE;
 }
 
+static avifBool avifParseContentLightLevelInformationBox(avifProperty * prop, const uint8_t * raw, size_t rawLen, avifDiagnostics * diag)
+{
+    BEGIN_STREAM(s, raw, rawLen, diag, "Box[clli]");
+
+    avifContentLightLevelInformationBox *clli = &prop->u.clli;
+
+    AVIF_CHECK(avifROStreamReadU16(&s, &clli->maxCLL));
+    AVIF_CHECK(avifROStreamReadU16(&s, &clli->maxPALL));
+
```

Comment:
Style Nit: Removing unnecessary blank line above the return.

Additionally, if (0,0) indicate the default/absent/ignore value, should this be considered an error if (0,0) is discovered here?

Reply:
MIAF 7.4.4.2.1 indicates that that zero values are permitted to indicate an unknown value, so I feel it should not be considered an error.